### ISSUE TITLE
If anybody has hosted this website...kindly share how did you did it.

### ISSUE THREAD

**Issue by Bdisha123** [safak/youtube23#32]

**Comment by amin123o9** [comment]
HOST IT USING NETLIFY.  vite built then copy the dev column and paste it in netlify

**Comment by Nitish-hack** [comment]
thats it I did these steps but not working

**Comment by amin123o9** [comment]
Is your project running on local host? If yes then it should work at any cost. If there is any error please tell me.

**Comment by Nitish-hack** [comment]
yeah I did tried some days back.............. did you deployed it  .. if yes can you share link to your website

**Comment by amin123o9** [comment]
ya sure here-https://amin123o9.netlify.app/

**Comment by niravpanchal22** [comment]
how to add react -mini folder in out project without clone?

**Comment by niravpanchal22** [comment]
@amin123o9  
how to add react -mini folder in out project without clone?
